k8s/helm/templates/frontend-server-service.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend-server
+  namespace: chroma
+spec:
+  ports:
+    - name: frontend-server
+      port: 8000
+      targetPort: 8000
+  selector:
+    app: frontend-server
+  type: LoadBalancer
```

Comment:
@beggers, wdyt about moving this to `values.yaml` and making it default to `ClusterIP`, if one runs this locally in Kind/minikube the service will be in a pending state as there is no external LB.